coral-hive/src/main/java/com/linkedin/coral/transformers/CoralRelToSqlNodeConverter.java

```diff
@@ -92,8 +92,36 @@ public boolean requireCastOnString() {
       }
 
       /**
-       * Override this method so that there will be explicit and correct table alias for selected fields, which is necessary for
-       * data type derivation on SqlNodes
+       * Override this method so that there will be explicit table alias for all the fields, which is necessary for
```

Comment:
`so that table alias is prepended to all field references (e.g., "table.column" or "table.struct.filed" instead of "column" or "struct.field")`.